CONTRIBUTING.md

```diff
@@ -16,7 +16,7 @@ You need a basic understanding of [Git and GitHub.com](https://guides.github.com
 * If your PR has the label 'cla-required' assigned, [complete the Contribution License Agreement (CLA)](https://cla.dotnetfoundation.org/).
 * Respond to PR feedback.
 
-For an example where this process led to publication of a new article, see [Issue &num;67](https://github.com/dotnet/docs/issues/67) and [Pull Request &num;798](https://github.com/dotnet/docs/pull/798) in the .NET Docs repository. The new article is [Documenting your code](https://docs.microsoft.com/dotnet/articles/csharp/codedoc).
+For an example where this process led to publication of a new article, see [Issue &num;1477](https://github.com/dotnet/docs/issues/1477) and [Pull Request &num;18955](https://github.com/dotnet/docs/pull/18955) in the .NET Docs repository. The new article is [Documenting your code](https://docs.microsoft.com/dotnet/articles/csharp/codedoc).
```

Comment:
```suggestion
For an example where this process led to publication of a new article, see [Issue &num;1477](https://github.com/dotnet/docs/issues/1477) and [Pull Request &num;18955](https://github.com/dotnet/docs/pull/18955) in the .NET Docs repository. The new article is [Use code coverage for unit testing](/dotnet/core/testing/unit-testing-code-coverage).
```

Reply:
Indeed @guardrex , somehow I've missed that, thank you.